moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2050,17 +2050,16 @@ double RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<
     {
       if (!joint->getType() == JointModel::PRISMATIC && !joint->getType() == JointModel::REVOLUTE)
       {
-        CONSOLE_BRIDGE_logError("Unsupported joint type %zu in JointModelGroup %s testJointSpaceJump can only support "
-                                "prismatic and revolute joints.",
-                                joint->getType(), group->getName().c_str());
-        throw Exception("Unsupported joint type");
+        CONSOLE_BRIDGE_logWarn("Unsupported joint type %s in JointModelGroup %s testAbsoluteJointSpaceJump can only "
```

Comment:
Please split the message into two separate sentences. Maybe even separated by a newline char `\n`.